src/arrayMethodJoin.js

```diff
@@ -5,7 +5,33 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
Use default parameter for separator
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```